portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,25 +35,38 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  // default values without user input
-  private ArrayList<String> messages = new ArrayList<String>(
-      Arrays.asList("No comments")
-  );
-
+  private ArrayList<Comment> messages = new ArrayList<>();
+  
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     
-    // Convert the arraylist to JSON
-    String json = convertToJson(messages);
+    Query query = new Query("Comment").addSort("timestamp_UTC", SortDirection.DESCENDING);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
 
-    // Send the JSON as the response
+    ArrayList<Comment> comment_lst = new ArrayList<>();
```

Comment:
Please use camelCase for variable name comment_lst, timestamp_UTC:
https://google.github.io/styleguide/javaguide.html#s5.2.7-local-variable-names